scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -135,11 +135,11 @@ class FossId internal constructor(
          */
         private fun queryAuthenticator(repoUrl: String): String {
             val repoUri = repoUrl.toUri().getOrElse {
-                log.warn { "Host cannot be extracted for $repoUrl." }
+                log.warn { "The repository '$repoUrl' in not a valid URI." }
```

Comment:
'in not' -> 'is not'

Consider to reduce it to "The repository '$repoUrl' is not valid.

Reply:
I went for yet another wording 😀